devices/schneider_electric.js

```diff
@@ -78,6 +78,24 @@ module.exports = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['NHROTARY/DIMMER/1'],
+        model: 'Rotary dimmer',
```

Comment:
Please add the model written on the box or device (rule of thumb is that is should be googleable). Rest of of the code looks good, what part do you still question?

Reply:
Sure! The model name differs depending on color but will add the name of the white standard one, WDE002334, which is what Schneider uses the most. Regarding the rest it was more a uncertainty from me, since I didnt find as many other examples using ballast config, but it works just fine locally when using my external converter.

Reply:
Good, I will merge this.